shell/platform/fuchsia/flutter/vulkan_surface_pool.cc

```diff
@@ -43,6 +43,11 @@ VulkanSurfacePool::VulkanSurfacePool(vulkan::VulkanProvider& vulkan_provider,
   sysmem_allocator_->SetDebugClientInfo(GetCurrentProcessName(),
                                         GetCurrentProcessId());
   FML_DCHECK(status != ZX_OK);
+
+  status = fdio_service_connect(
```

Comment:
We should only connect to this if the session is null, to avoid disturbing production code as much as possible

Specifically, fuchsia tests failed when reveman tried to add Allocator support before and it got rolled back, so I want to be very cautious here

Reply:
Done.